cluster-autoscaler/core/scale_up.go

```diff
@@ -605,7 +605,7 @@ func filterNodeGroupsByPods(
 	return result
 }
 
-func executeScaleUp(context *context.AutoscalingContext, clusterStateRegistry *clusterstate.ClusterStateRegistry, info nodegroupset.ScaleUpInfo, gpuType string, now time.Time) errors.AutoscalerError {
+func executeScaleUp(context *context.AutoscalingContext, clusterStateRegistry *clusterstate.ClusterStateRegistry, info nodegroupset.ScaleUpInfo, resourceName, gpuType string, now time.Time) errors.AutoscalerError {
```

Comment:
nit: I'd add the gpu prefix to the parameter name as well, it's ambiguous without it. If you're changing that, `metricResourceName` x3 in this PR would probably be better as `gpuResourceName` as well (but that's much less ambiguous since they're returned from a function with GPU in the name and immediately used).

Reply:
done